spec/unit/speak_slack_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../../app/speak_slack'
+require_relative '../../constants'
+
+module FoodTruck
+  RSpec.describe SpeakSlack do
+    describe 'SpeakSlack says' do
+      context 'when passed an empty array' do
+        it 'food trucks cannot be found' do
```

Comment:
"it food trucks cannot be found" is awkward English.  How about `it "returns a string indicating no food trucks can be found"`?